core/domain/skill_domain.py

```diff
@@ -653,3 +653,36 @@ def to_dict(self):
             'skill_model_created_on': self.skill_model_created_on,
             'skill_model_last_updated': self.skill_model_last_updated
         }
+
+
+class UserSkillMastery(object):
+    """Domain object for a user's mastery of a particular skill."""
+
+    def __init__(self, model_id, user_id, skill_id, degree_of_mastery):
```

Comment:
This method should have a dedicated unit test in skill_domain_test.

Reply:
okay